### DISCUSSION TITLE
Quit and remove app?

### DISCUSSION THREAD

**Discussion by switz** [johnlindquist/kit#127] [Q&A] [answered]
Honestly a pretty cool little app, but I don't think I quite have a need for it at the moment. Any advice on quitting and removing the app? It auto-restarts on pkill.

**Comment by johnlindquist** [comment, marked as answer]
That must be the auto-updater misbehaving and thinking a relaunch is needed. "Quit" is available in the main menu under the "Other" tab. I've always been able to end the process from the activity monitor when bugs pop up.

Worst case, it shouldn't launch on startup, so there's always restart then trash.

**Reply by switz** [reply]
Ah I didn't realize there was a quit under that tab. Thanks! Again, awesome app, just not something I need.
